src/styles/GlobalStyles.jsx

```diff
@@ -15,6 +15,8 @@ const GlobalStyles = createGlobalStyle`
     --light-grey: #bababa;
     --dark-grey: #606060; /* this is not used anywhere */
     --off-white: #f5f5f5;
+    --nav-background-color: white;
```

Comment:
We can just call this "white". It's likely we'll use this color in multiple places.